src/components/useStore/useStore.js

```diff
@@ -32,6 +48,22 @@ const useStore = create(
 		updateBookmark: bookmarkStatus => {
 			set({ bookmarkStatus });
 		},
+
+		controlEdit: (id, comment) => {
+			set(state => {
+				return {
+					newBooks: state.newBooks.map(books =>
+						books.id === id
+							? {
+									...books,
+
+									comment,
+							  }
+							: books
+					),
```

Comment:
Same here :)

Reply:
Same as above

Reply:
No problem. You can mention this issue with the coaches on occasion.
Approved.